packages/blockchain-api/src/events/ExchangeCeloToToken.ts

```diff
@@ -28,7 +29,7 @@ export class ExchangeCeloToToken extends TransactionType {
       transaction,
       inTransfer,
       outTransfer,
-      this.context.token,
+      this.context.token || CUSD,
```

Comment:
@carterqw2 could you confirm this doesn't have unintended effects?

Reply:
I believe that we'll have to rewrite the whole exchange thing when we start supporting cEUR (soon) but now it looks safe.